SparseGrids/Benchmarks/benchCommon.hpp

```diff
@@ -9,14 +9,16 @@ enum BenchFuction{
     bench_none,
     bench_make,
     bench_loadneeded,
-    bench_evaluate
+    bench_evaluate,
+    bench_fft
 };
 
 BenchFuction getTest(std::string const &s){
     std::map<std::string, BenchFuction> str_to_test = {
         {"evaluate", bench_evaluate},
         {"loadneeded", bench_loadneeded},
         {"makegrid", bench_make},
+        {"fft", bench_fft}
```

Comment:
* you have the benchmark changes in both commits, you will have to drop it from this one